### ISSUE TITLE
Invalid URL in h.alerts.slack()

### ISSUE THREAD

**Issue by muhyuddin** [bamhm182/SynackAPI#11]
Using below code to test slack notification:

```
h = synack.Handler()
h.db.synack_url = "https://hooks.slack.com/services/...."
h.alerts.slack("Hiii")
```

And it is throwing the error:
```
requests.exceptions.MissingSchema: Invalid URL 'None': No schema supplied. Perhaps you meant http://None?
```

Have tried to debug but can't understand the code in Plugin.py.

**Comment by muhyuddin** [comment]
I guess the problem is, Db instance is being initiated in Alerts calss as well,  thats why all config being clear/ set to default.
at last line: 
```
class Alerts(Plugin):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        for plugin in ['Db']:
            print(plugin)
            setattr(self,
                    plugin.lower(),
                    self.registry.get(plugin)(self.state))
```
**setattr(self, plugin.lower(), self.registry.get(plugin)(self.state))**

**Comment by bamhm182** [comment]
Just noticed the documentation there has a typo. It should be `slack_url`, not `synack_url`.

**Comment by bamhm182** [comment]
This was fixed in commit c97c09b44333489aee7eb03fa8cc15cff8127e50. Closing.
